packages/vee-validate/src/Form.ts

```diff
@@ -75,6 +75,10 @@ const FormImpl = /** #__PURE__ */ defineComponent({
       type: Boolean,
       default: false,
     },
+    name: {
+      type: String,
+      default: undefined,
```

Comment:
Shouldn't this be `default: "Form"`?

Reply:
it can be, but we already default it to "Form" in the `useForm` composable.

defaulting it here as well, would be redundant I think.

Reply:
Yep, no need to default it twice. I don't have a strong opinion here.